lib/matplotlib/lines.py

```diff
@@ -759,9 +759,8 @@ def draw(self, renderer):
                 gc = renderer.new_gc()
                 self._set_gc_clip(gc)
 
-                ln_color_rgba = self._get_rgba_ln_color()
-                gc.set_foreground(ln_color_rgba, isRGBA=True)
-                gc.set_alpha(ln_color_rgba[3])
+                lc_rgba = mcolors.to_rgba(self._color, self._alpha)
```

Comment:
OK, so `_alpha` overrides the alpha in `self._color` if its not `None`.  I'm confused if thats what happened before.

Reply:
Yes.  As a general rule, an explicit alpha will override one that comes "with the color" (we can always discuss whether that's a good strategy (I think it's a mess) but that's what we have for now).  The exception is the one noted in the comment re: markeredgecolor.
(Before, that happened via get_rgba_ln_color, which I had forgotten to remove but now did.)